### PR TITLE
Faz-World Sync 04/22/2022

### PR DESCRIPTION
**Pull request by noelle-lavenza**
## Description of changes

Ports a number of fixes from downstream.
## Why and what will this PR improve

Various bugfixes from downstream.

## Authorship
Me.

## Changelog

:cl:
bugfix: Fixed prebuilt floor lights starting with the wrong icon/status.
bugfix: Fixed base wood tables and chairs not having materials set.
bugfix: Fixed an improper usage of DEFAULT_WALL_MATERIAL in place of "steel".
bugfix: Made shuttle windows block air
bugfix: Fixed things without hearts bleeding/making blood splatters
bugfix: Fixed "something something on their hands" message
bugfix: Fixed runtimes with species cultural info and appearance being called before organs
bugfix: Fixes MMI installation
bugfix: Fixes certain limbs not being set up properly on spawn
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -217,6 +217,9 @@ meteor_act
 	if(W.damtype != BRUTE)
 		return
 
+	if(!should_have_organ(BP_HEART))
+		return
```
Comment (afterthought2) [inline code/modules/mob/living/carbon/human/human_defense.dm:221]: Is this a good proxy for bloodiness across commonly used species? I seem to recall mechanical species using blood overlays to hack in oil spills and stuff.

Comment (MistakeNot4892) [reply code/modules/mob/living/carbon/human/human_defense.dm:221]: It's a standard check for other species code involving blood, dating back to one of the foundational organ PRs. Should be changed to its own proc most likely but that would be a broader refactor.

Comment (afterthought2) [reply code/modules/mob/living/carbon/human/human_defense.dm:221]: fair enough.

**Review by afterthought2 — APPROVED** [review]

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 1 above)
Comment (noelle-lavenza) [reply code/modules/mob/living/carbon/human/human_defense.dm:221]: Yeah, I used it specifically because other similar code did.

**Pull request merged**